runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -244,11 +244,24 @@ default FlashNotification flashNotification()
 		return FlashNotification.DISABLED;
 	}
 
+	@ConfigItem(
+		keyName = "flashNotificationDuration",
+		name = "Notification duration",
+		description = "Duration of flash notification. Value of 0 will force notification to wait until cancelled",
```

Comment:
This controls the duration of both flash and solid notification types.
```suggestion
		description = "Duration of notification overlay. A value of 0 will force notification to wait until cancelled",
```